test/test.js

```diff
@@ -814,3 +814,10 @@ test('without --dot option exclude folders/files with a dot', async t => {
   t.is(result.stdout, '');
   t.is(result.stderr, '');
 });
+
+test('with --quiet option does not print to stdout', async t => {
```

Comment:
Please check for failed exit code

Reply:
The exit code is wrong here. There are linting errors, but the exit code is 0. It should have been set to 1 in printResult, but this change skipped that call. It may be necessary to duplicate the (results.length > 0) check here.